string path
---
std_msgs/Empty empty
